src/components/numeric-badge/index.style.js

```diff
@@ -0,0 +1,14 @@
+const { css } = require('@emotion/css');
+
+export const badge = css`
+  color: #333;
+  font-weight: 500;
+  font-size: 0.7rem;
+  border-radius: 11px;
```

Comment:
not necessarily anything to be fixed here, but just an fyi that if you're using a mix of rems and px's your page may be broken in some cases

Reply:
Yes. Thank you for pointing out. I am planning to resolve this in the future